creme/billing/forms/export.py

```diff
@@ -84,6 +84,7 @@ def get_context(self, name, value, attrs):
             #     'autocomplete': True,
             # },
             attrs=field_attrs,
+            is_required=True,
```

Comment:
Hum I did not understand that the default behaviour has changed (you talk about the JavaScript component DynamicSelect).
Or we explain better how people have to fix their code, or we keep the old behaviour as default.